tensorflow_serving/batching/basic_batch_scheduler.h

```diff
@@ -155,15 +155,14 @@ class BasicBatchScheduler : public BatchScheduler<TaskType> {
 
     // The number of threads to use to process batches.
     //
-    // This parameter also determines the maximum number of batches worth of
-    // tasks that can be enqueued: If all batch threads are busy, Schedule()
-    // will accept at most 'num_batch_threads' additional batches worth of tasks
-    // before rejecting tasks with an UNAVAILABLE error.
+    // If all threads are busy, Schedule() will enqueue up to 
+    // max_enqueued_batches * max_batch_size individual tasks before throwing
```

Comment:
Actually that's only true if tasks are unit size (all task have size 1). That is indeed a common case, but it is not universal.

I would prefer dropping this comment. This information is covered above anyway.

Also, I would prefer to keep the documentation the same as the corresponding part of shared_batch_scheduler. Let's make the corresponding comment blocks identical, shall we?